src/_includes/banner.html

```diff
@@ -5,6 +5,6 @@
     https://github.com/flutter/website/issues/3897
   {% endcomment %}
   <div class="site-banner site-banner--default" role="alert">
-    Free access to best-selling book Flutter Apprentice, for a limited time only! <a href="/apprentice-giveaway">Learn more...</a>
+    The next Flutter challenge is here! Join the Flutter Puzzle Hack for a chance to win prizes.<a href="/apprentice-giveaway">Learn more...</a>
```

Comment:
What should this link be? Maybe just put the link on [Flutter Puzzle Hack]?

Reply:
```suggestion
    The next Flutter challenge is here! Join the <a href="https://flutterhack.devpost.com/">Flutter Puzzle Hack</a> for a chance to win prizes.
```